starters/qwik-graphql-tailwind/src/components/data-fetching/data-fetching.tsx

```diff
@@ -71,19 +71,11 @@ export const DataFetching = component$(() => {
 });
 
 export async function fetchGreeting(greeting: string, abortController?: AbortController): Promise<HelloResponse> {
-  const { executeQuery$ } = useQuery(GET_GREETING);
+  const resp = await useStarterQuery(GET_GREETING, { greeting }, abortController?.signal);
 
   if (!greeting) {
     greeting = '';
   }
```

Comment:
I don't understand why this if() is after the API call.

Reply:
Idk at all but this is not a change related to this PR @AllStackDev1

Reply:
Ok, but I think the logic has been defeated here because it was meant to make sure we don't have undefined or null values passed.